website/addons/base/views.py

```diff
@@ -571,7 +571,6 @@ def addon_deleted_file(auth, node, error_type='BLAME_PROVIDER', **kwargs):
         'file_path': file_path,
         'file_name_title': file_name_title,
         'file_name_ext': file_name_ext,
-        'version_id': version.identifier,
```

Comment:
As @mfraezz indicated, this was failing because deleted files don't have a version, so when it tried to access version.identifier, there was an error. This is _proooobably_ okay, but another way to have fixed it would have been to do `'version_id': None,` instead, which would keep the option serialized but ensure that it was null when it made it to the outside world.

All that being said, I don't think anything's using the version field on a deleted file, although I also don't know who would be using `addon_deleted_file()` at all. Have you gone through and found out what uses `addon_deleted_file()` and done manual testing to ensure that it won't be problematic?